operations/template/variables.tf

```diff
@@ -19,3 +19,8 @@ variable "alert_slack_email" {
   nullable  = false
   sensitive = true
 }
+
+variable "service_health_locations" {
+  type    = string
+  default = "global"
+}
```

Comment:
Do we need this?  Do we actually customize this to anything?  I feel we could just hardcode `"global"` in the `azurerm_monitor_activity_log_alert`.

Reply:
Removed variable for now as its only used in this file. If more instances pop up, will refactor later.